src/key.cpp

```diff
@@ -275,6 +275,13 @@ bool CKey::SignSchnorr(const uint256& hash, Span<unsigned char> sig, const uint2
         if (!secp256k1_keypair_xonly_tweak_add(GetVerifyContext(), &keypair, tweak.data())) return false;
     }
     bool ret = secp256k1_schnorrsig_sign(secp256k1_context_sign, sig.data(), hash.data(), &keypair, aux ? (unsigned char*)aux->data() : nullptr);
+#ifdef ENABLE_SCHNORR_VERIFY
+    if(ret){
```

Comment:
Style nit: spaces before and after `(ret)`.

Reply:
resolved, thanks